src/jaxsim/rbda/contacts/common.py

```diff
@@ -304,30 +291,17 @@ def zero_state_variables(cls, model: js.model.JaxSimModel) -> dict[str, jtp.Arra
 
         return {}
 
-    def initialize_model_and_data(
-        self,
-        model: js.model.JaxSimModel,
-        data: js.data.JaxSimModelData,
-        validate: bool = True,
-    ) -> tuple[js.model.JaxSimModel, js.data.JaxSimModelData]:
+    @property
+    def _parameter_class(cls) -> type[ContactsParams]:
```

Comment:
Nit

```suggestion
    def _parameters_class(cls) -> type[ContactsParams]:
```

Reply:
Done with rebase in https://github.com/ami-iit/jaxsim/pull/299/commits/da62dffb32453a3c6953fd9ca4f73d1125ec03e9. Thanks!